app/components/Dashboard/MarketsRow.jsx

```diff
@@ -101,12 +101,8 @@ class MarketsRow extends React.Component {
 
         function getImageName(asset) {
             let symbol = asset.get("symbol");
-            if (
-                symbol === "OPEN.BTC" ||
-                symbol === "GDEX.BTC" ||
-                symbol === "RUDEX.BTC"
-            )
-                return symbol;
+            if (symbol === "OPEN.BTC" || symbol === "GDEX.BTC") return symbol;
+            if (symbol.startsWith("RUDEX.")) return symbol;
```

Comment:
This would remove the prepending RUDEX before all the RUDEX.* assets, I would not think that is appropriate.